sources/bloodhound/src/lib.rs

```diff
@@ -13,7 +13,7 @@ pub fn look_for_string_in_file(path: &str, search_str: &str) -> Result<bool, io:
         .any(|line| line.unwrap_or_default().contains(search_str)))
 }
 
-/// Reads a file and checks if the given `search_str` is present in its contents.
+/// Reads a file and checks if the given `search_strs` are present in its contents.
```

Comment:
nit: seems like the updates in this file were included in the wrong commit.

Reply:
This was a typo fix, and below was an evolution of the function based on the needs of this particular check.